pyseir/cdc/output_mapper.py

```diff
@@ -0,0 +1,526 @@
+
+    Attributes
+    ----------
+    model: SEIRModel
+        SEIR model with MLE parameter estimates.
+    forecast_given_time_range: callable
+        Makes forecast of a target during the forecast time window given a
+        model's prediction of target and model's t_list (t steps since the
+        inferred starting time of the epidemic, t0).
+    targets: list(Target)
+        List of Target objects.
+    forecast_time_units: list(ForecastTimeUnit)
+        List of ForecastTimeUnit objects, determines whether forecast target
+        is aggregated by day or week. Currently the mapper only supports
+        daily forecast.
+    forecast_uncertainty: ForecastUncertainty
+        Determines how forecast uncertainty is adjusted based on number of
+        days the forecast is made ahead and total days of observations.
+        Should be interpretable by ForecastUncertainty. Currently supports:
+        - ForecastUncertainty.DEFAULT: no adjustment
+        - ForecastUncertainty.NAIVE: rescale the standard deviation by factor (1
+                                     + days_ahead  ** 0.5)
+    result: pd.DataFrame
+        Output that meets requirement for CDC model ensemble submission for
+        given FIPS code.
+        Contains columns:
+        - forecast_date: datetime.datetime
+          the date on which the submitted forecast data was made available in
+          YYYY-MM-DD format
+        - target: str
+          Name of the forecast target, with format "<day_num> day ahead
+          <target_measure>" where day_num is number of days since
+          forecast_ate to each date in forecast time range.
+        - target_end_date: datetime.datetime
+          last date of forecast in YYYY-MM-DD format.
+        - location: str
+          2 digit FIPS code.
+        - type: str
+          "quantile" or "point"
+        - quantile: str
+          quantiles of forecast target measure, with format 0.###.
+        - value: float
+          Value of target measure at given quantile and forecast date for
+          given location.
+        - location_name: str
+          Name of the state.
+
+
+    Parameters
+    ----------
+    fips: str
+        State or County FIPS code
+    N_samples: int
+        Number SEIR model parameter sets of sample.
+    targets: list(str)
+        Names of the targets to forecast, should be interpretable by Target.
+    forecast_date: datetime.date
+        Date when the forecast is done, default the same day when the mapper
+        runs.
+    next_epi_week: epiweeks.week
+        The coming epi weeks, with the start date of which the forecast time
+        window begins.
+    forecast_time_units: list(str)
+        Time units of the forecast target, should be interpretable by
+        ForecastTimeUnit, currently the mapper only supports unit 'day'.
+    quantiles: list(float)
+        Values between 0-1, which are the quantiles of the forecast target
+        to collect. For default value check QUANTILES.
```

Comment:
> For default value check QUANTILES.

I don't understand what this means

Reply:
It means that global variables QUANTILES contains the default values.

Reply:
I will make this clearer in the comment.